erpnext/shopping_cart/cart.py

```diff
@@ -86,6 +85,28 @@ def place_order():
 				if item.qty > item_stock.stock_qty[0][0]:
 					throw(_("Only {0} in Stock for item {1}").format(item_stock.stock_qty[0][0], item.item_code))
 
+	#if checkout without payment has been enabled, submit the quotation, and convert to sales order
+	if cart_settings.sales_team_order_without_payment:
+	
+		quotation.save(ignore_permissions=True)
+		quotation.submit()
+		frappe.db.commit()
+
+		from erpnext.selling.doctype.quotation.quotation import _make_sales_order
+		sales_order = frappe.get_doc(_make_sales_order(quotation.name, ignore_permissions=True))
+		sales_order.payment_schedule = []
+		sales_order.transaction_date = nowdate()
+		sales_order.delivery_date = add_days(nowdate(),10)
```

Comment:
Should we make this a setting under shopping cart settings or add a 'Requested delivery Date' in the checkout form?

Reply:
I think the change in shopping cart settings is a better idea, ideally it would be easier if the seller configured the delivery period